src/Stan/Report/Html.hs

```diff
@@ -16,6 +16,7 @@ module Stan.Report.Html
 
 import Clay (compact, renderWith)
 import Data.Char (toLower)
+import Html
```

Comment:
Temporary import to make conflict resolution easier. But feel free to make import for the `Html` module implicit :slightly_smiling_face: I feel like these libraries (`type-of-html` and `clay`) require enormous imports because of how they were designed...